tests/happy-path.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test, expect, _electron as electron } from '@playwright/test';
+import { mkdtemp, rm } from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import { sep, join } from 'node:path';
+
+let installDir: string;
+
+
+test.beforeAll(async () => {
+  try {
+    installDir = await mkdtemp(`${tmpdir()}${sep}drupal-temp-`);
+    console.log(`Temporary directory created at: ${installDir}`);
+  } catch (err) {
+    console.error('Error creating temporary directory:', err);
+  }
+
+  console.log('Before tests');
+});
+
+test.afterAll(async () => {
+  try {
+    rm(installDir, { recursive:true } );
```

Comment:
Same idea here -- we can remove the `try/catch` safeguards and let this crap out with an error if the delete fails.